Inferences/ForwardDemodulation.cpp

```diff
@@ -58,8 +58,7 @@ void ForwardDemodulation::attach(SaturationAlgorithm* salg)
 	  _salg->getIndexManager()->request(DEMODULATION_LHS_CODE_TREE) );
 
   _preorderedOnly = getOptions().forwardDemodulation()== Options::Demodulation::PREORDERED;
-  _redundancyCheck = getOptions().demodulationRedundancyCheck() != Options::DemodulationRedunancyCheck::OFF;
-  _encompassing = getOptions().demodulationRedundancyCheck() == Options::DemodulationRedunancyCheck::ENCOMPASS;
+  _redundancyCheck = getOptions().demodulationRedundancyCheck();
```

Comment:
At some future point this could just be a single call to `env.options->demodulationRedundancyCheck()`, I doubt it's worth keeping the state around. Not now, however.